_data/simple-icons.json

```diff
@@ -2368,6 +2368,11 @@
             "hex": "D33C43",
             "source": "https://cakephp.org/logos"
         },
+        {
+            "title": "Calendly",
+            "hex": "016AFF",
```

Comment:
```suggestion
            "hex": "006bff",
```

Reply:
@PeterShaggyNoble - this needs to be uppercase, so `#006BFF`